src/Button/Button.Styled.js

```diff
@@ -0,0 +1,173 @@
+import styled from 'styled-components';
+
+
+export const StyledButton = styled.button`
+    color: #F6F9FC;
+    font:14px 'Arial';
+    border: none;
+    background-color:#FF4E54;
+    padding:11px 28px;
+    margin:10px;
+    cursor: pointer;
+`;
+export const SecondButton = styled.button`
+    color: #4a4a68;
+    margin:10px;
+    border: none;
+    font:14px 'Arial';
+    background-color:#F3F4F6;
+    padding:11px 28px;
+    cursor: pointer;
+`;
+export const ThridButton=styled.button`
+    color: #f6f9fc;
+    margin:10px;
+    border: none;
+    font:14px 'Arial';
+    background-color:#4CAF50;
+    padding:11px 28px;
+    cursor: pointer;
+`;
+export const FourthButton=styled.button`
+    color: #f6f9fc;
+    margin:10px;
+    border: none;
+    font:14px 'Arial';
+    background-color:#ff9800;
+    padding:11px 28px;
+    cursor: pointer;
+`;
+export const Primarybuttonhover=styled.button`
+    border:1px solid black;
+    color: #FF4E54;
+    font:14px 'Arial';
+    border: none;
+    background-color:#f6f9fc;
+    padding:11px 28px;
+    margin:10px;
+    cursor: pointer;
+    &:hover,
+    active{
+        background-color:#FF4E54;
+        color:#f6f9fc;
+    }
+`
+export const SecondButtonhover = styled.button`
+    color: #4a4a68;
+    margin:10px;
+    border: none;
+    font:14px 'Arial';
+    background-color:#f6f9fc;
+    padding:11px 28px;
+    cursor: pointer;
+    &:hover,
+    active{
+        background-color:#4a4a68;
+        color:#f6f9fc;
+    }
+`;
+export const ThridButtonhover=styled.button`
+    color: #4CAF50;
+    margin:10px;
+    border: none;
+    font:14px 'Arial';
+    background-color:#f6f9fc;
+    padding:11px 28px;
+    cursor: pointer;
+    &:hover,
+    active{
+        background-color:#4CAF50;
+        color:#f6f9fc;
+    }
+`
+export const FourthButtonhover=styled.button`
+    color: #ff9800;
+    margin:10px;
+    border: none;
+    font:14px 'Arial';
+    background-color:#f6f9fc;
+    padding:11px 28px;
+    cursor: pointer;
+    &:hover,
+    active{
+        background-color:#4CAF50;
+        color:#f6f9fc;
+    }
+`
+
+
+
+export const Save_hover=styled.button`
+  border: 2px solid; // Add a unit (e.g., pixels)
```

Comment:
@KBhautik6 add color for border from theme

Reply:
okay